### PR TITLE
Support/2.5

### PR DESCRIPTION
**Pull request by purplegrape**
chinese support is poor for a long time，(maybe since itop 2.0)
so I update my own offline translation（itop 2.5.0） for the moment
request to merge to 2.5 branch  if no error occurred

### CODE REVIEW

**Comment by piRGoif** [comment]
Hello, 
Many thanks for this big work !
Would you mind squash all of those commits into one ?

**Comment by purplegrape** [comment]
I will try it later

File: datamodels/2.x/itop-attachments/zh.dict.itop-attachments.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	'Attachments:History_File_Removed' => '附件 %1$s 已移除.',
+	'Attachments:AddAttachment' => '添加附件: ',
+	'Attachments:UploadNotAllowedOnThisSystem' => '本系统不支持文件上传.',
+	'Attachment:Max_Go' => '(最大文件尺寸: %1$s GB)',
+	'Attachment:Max_Mo' => '(最大文件尺寸: %1$s MB)',
+	'Attachment:Max_Ko' => '(最大文件尺寸: %1$s KB)',
+	'Attachments:NoAttachment' => '没有附件. ',
+	'Attachments:PreviewNotAvailable' => '该附件类型不支持预览.',
+));
+?>
```
Comment (piRGoif) [inline datamodels/2.x/itop-attachments/zh.dict.itop-attachments.php]: New file not already translate

File: datamodels/2.x/itop-backup/zh.dict.itop-backup.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	'bkp-next-backup' => '下一次备份将发生在 <b>%1$s</b> (%2$s) 的 %3$s',
+	'bkp-button-backup-now' => '立即备份!',
+	'bkp-button-restore-now' => '还原!',
+	'bkp-confirm-backup' => '请确认是否立即开始备份.',
+	'bkp-confirm-restore' => '请确认要还原的备份文件是 %1$s.',
+	'bkp-wait-backup' => '请等待备份完成...',
+	'bkp-wait-restore' => '请等待还原完成...',
+	'bkp-success-restore' => '还原成功.',
+));
+?>
```
Comment (piRGoif) [inline datamodels/2.x/itop-backup/zh.dict.itop-backup.php]: New translation

File: datamodels/2.x/itop-change-mgmt/zh.dict.itop-change-mgmt.php [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+	'Class:Change/Stimulus:ev_finish+' => '',
+	'Class:Change/Attribute:outage' => '停机',
+	'Class:Change/Attribute:outage+' => '',
+	'Class:Change/Attribute:outage/Value:no' => '否',
+	'Class:Change/Attribute:outage/Value:no+' => '',
+	'Class:Change/Attribute:outage/Value:yes' => '是',
+	'Class:Change/Attribute:outage/Value:yes+' => '',
+));
+
+?>
```
Comment (piRGoif) [inline datamodels/2.x/itop-change-mgmt/zh.dict.itop-change-mgmt.php]: New translation file

**Comment by piRGoif** [comment]
I've just squash & merge your commits, thanks !
The target branch was support/2.5, this will be merged to release/2.6 tonight (french time).

**Comment by piRGoif** [comment]
I began to review the numerous PR you created yesterday (#29, #30, #31, #32, #33, #34, #35), they seem to contain most of the modifications you submitted in this PR, with a few differences...
Can you please submit in just one PR, and with the fewest number of commits possible, only the keys that were not already present in this PR ?

**Comment by purplegrape** [comment]
I've already  found there's some difference of translation between 2.5 and 2.6
my origin plan is maintain 2.5 and 2.6 separately

I don't know too much about  your development plan 

If you really wanna merge 2.5 to 2.6 , I have to close #29 to #35 then back to work on 2.5 ,
so that every time you merge 2.5 to 2.6 , will get the latest chinese translation

am I right ?

**Comment by piRGoif** [comment]
Sorry, contributing is not well documented for now, but we are working on some documentation that will be out in the upcoming weeks !
A CONTRIBUTING.md has already been commited in release/2.6, it will soon be merged to develop. We are also working on some more detailed info in our public wiki.

In the meantime, I'm going to merge support/2.5 to release/2.6 now - there are some merge conflicts so it will be pushed in some minutes :)

The release/2.6 branch has some new dictionary keys because this version brings lots of fixes and enhancements. If you already translate all of the 2.5 keys, you should continue on the release/2.6 branch to translate the new keys - wait for the merge to be pushed though.

Thanks !

**Comment by piRGoif** [comment]
Merge not done yet, because fixes are still working on... Will be done on monday morning !

**Comment by piRGoif** [comment]
Hello,
I've done the merge (support/2.5 to release/2.6) and update the translations : you can work on translations on release/2.6 branch starting on 80a8f7a888a8c12110d2442fc96431de0a5af49f.

**Pull request merged**